README.rst

```diff
@@ -73,6 +73,13 @@ Your AWS credentials must be present in ``~/.aws/credentials``. sops uses aws-sd
 	aws_access_key_id = AKI.....
 	aws_secret_access_key = mw......
 
+On the other hand you can also declare the ``AWS_ACCESS_KEY_ID`` and ``AWS_SECRET_ACCESS_KEY`` in your environment variables and sops will also automatically read it as mentioned in the issue `#432 <https://github.com/getsops/sops/issues/432>`_.
+
+.. code:: bash
+
+	export AWS_ACCESS_KEY_ID="AKI......"
+	export AWS_SECRET_ACCESS_KEY="mw......"
```

Comment:
Instead of referring to the issue and/or providing examples, it may actually be better to refer to the documentation of the SDK itself which outlines all possible options: https://aws.github.io/aws-sdk-go-v2/docs/configuring-sdk/#specifying-credentials